rele/config.py

```diff
@@ -102,5 +102,10 @@ def load_subscriptions_from_paths(sub_module_paths, sub_prefix=None, filter_by=N
             if filter_by and not subscription.filter_by:
                 subscription.set_filters(filter_by)
 
-            subscriptions.append(subscription)
-    return subscriptions
+            if subscription.name in subscriptions:
+                raise RuntimeError(
+                    f"Duplicated subscription found: {subscription.name}."
+                )
```

Comment:
It would be usefule to also log the names of the two subscription `functions` that collide

Reply:
Done it in https://github.com/mercadona/rele/pull/187/commits/3b3256050aa13935c878f076df358550b00d8a89